### PR TITLE
Insert a Help Button into custom UI provided to WordPressAuthenticator

### LINKED ISSUE #3124: [UL&S] Activate WPAuthenticator flag for Unified Login
Pass `enableUnifiedAuth` and `enableUnifiedCarousel` to `WordPressAuthenticatorConfiguration` as true.

This will make WPAuthenticator display the UL&S flow.

This issue is not to be merged into `develop`, but into a long lived feature branch.

There is some indetermination here about the actual scope of the task. I think an acceptable scope here is that the app builds, and it is possible to login, even if the flow does not look like it should

### PR DESCRIPTION
**Pull request by ctarda**
Closes #3124 

⚠️ Please note this PR is not against develop, but against the feature branch for UL&S 🙇

The actual work of inserting the Help button is done by WPAuthenticator, and implemented in this PR: wordpress-mobile/WordPressAuthenticator-iOS#527

What this PR does is point WPAuthenticator to the relevant branch and comment out a bit of code that will have to be un-commented when PR #3186 is merged.

| Before | After |
| ---- | ---- |
|  |  |

Notice the Help button in the navigation bar

## Changes
* Point the reference to WPAuthenticator to the branch that implements the solution
* Comment out (temporarily, only until #3186 is merged) two lines of code that would break the build. The reason for that is that PRs in WPAuthenticator that break the API are not in sync with PRs in WooCommerce.

## Testing
* Checkout the branch
* Run `bundle exec pod install`
* Try to log in with a Site Address, using a site that does not have Jetpack installed, or does not have Jetpack connected (as this is the only custom error we present)

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift [hunk 1]
```diff
@@ -49,8 +49,10 @@ struct JetpackErrorViewModel: ULErrorViewModel {
     }
 
     func didTapSecondaryButton(in viewController: UIViewController?) {
-        let refreshCommand = NavigateToRoot()
-        refreshCommand.execute(from: viewController)
+        // To be removed when PR #3186 is merged.
```
Comment (ctarda) [inline WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift]: This should not be part of the diff, it is temporary, and I will remove it before merging. 

But, the API of WPAuthenticator is a bit ahead

Comment (jaclync) [reply WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift]: Sounds tricky with different changes going on in both repos!

**Review by jaclync — APPROVED** [review]
Code changes look good! I tried testing with a new Pressable site with Jetpack deactivated but I still see the login by credentials screen 🤔 (I couldn't delete Jetpack plugin on my site) not sure if I missed some setup for the error screen?


plugins | after logging in with site address
-- | --
Screen Shot 2020-11-30 at 6 12 15 PM | Simulator Screen Shot - iPhone 11 Pro - 2020-11-30 at 18 13 42

**Review by Garance91540 — APPROVED** [review]
Hi Cesar, this is looking good! You probably will address this separately, but I wanted to make sure that all the CTAs ("Back", "Help", "Find your store address") will be displayed in Pink 50 https://d.pr/i/RsNXke

**Comment by ctarda** [comment]
Thanks for the review!

I logged #3251 for setting the color of the CTAs properly, and #3252 for reviewing the flow for testable sites. 

I'll merge as soon as CI gives the ✅

**Pull request merged**